### DISCUSSION TITLE
Possibly disable in-app announcements for sponsored installs

### DISCUSSION THREAD

**Discussion by SoCuul** [massCodeIO/massCode#457] [General]
It would be nice if the in-app announcements (about other projects) were not present in the sponsored executables

**Comment by antonreshetov** [comment]
Sponsorship does not provide any benefits other than disabling the Unsponsored label. In general, sponsorship is a tribute to work that has already been done.

**Comment by SoCuul** [comment]
> Sponsorship does not provide any benefits other than disabling the Unsponsored label. In general, sponsorship is a tribute to work that has already been done.

It's just unfortunate that while I love the software, I hate having unnecessary notifications which have nothing to do with the functionality of the app.

**Reply by antonreshetov** [reply]
Well, there is a lot of unnecessary things around us, as it seems to everyone.

You always have a way out, because this is an open source application, you can make your own build, fork, etc.
